released/packages/coq-rupicola/coq-rupicola.0.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: [
+  "Massachusetts Institute of Technology"
```

Comment:
Please use the actual list of authors

Reply:
This is the actual list of authors according to https://github.com/mit-plv/rupicola/blob/master/AUTHORS.  Do you mean the list of contributors at https://github.com/mit-plv/rupicola/blob/master/CONTRIBUTORS ?

Reply:
Oh, we should fix these files.  The list should be myself, jade, dustin, andres, and adam.

Reply:
Could you use code suggestion to suggest formatting for the list, or update the files and I'll copy over their contents here?

Reply:
I don't know how to do a code suggestion that spans multiple lines :/ Here's the list, though:

Clément Pit-Claudel <clement.pitclaudel@live.com>
Jade Philipoom
Dustin Jamner
Andres Erbsen
Adam Chlipala